QualifiedTeachersApi/src/QualifiedTeachersApi/DataStore/Crm/DataverseAdapter.SetIttResultForTeacher.cs

```diff
@@ -251,6 +251,9 @@ string GetDescription()
 
         public async Task<SetIttResultForTeacherLookupResult> LookupData()
         {
+            var getAllEytsTeacherStatusesTask = _dataverseAdapter.GetAllEarlyYearsStatuses();
```

Comment:
Ideally we should be using `RequestBuilder` here (for all the methods that have overloads that take one).